Agrume/AgrumeCell.swift

```diff
@@ -52,15 +52,25 @@ final class AgrumeCell: UICollectionViewCell {
   private var imageDragOffsetFromActualTranslation: UIOffset!
   private var imageDragOffsetFromImageCenter: UIOffset!
   private var attachmentBehavior: UIAttachmentBehavior?
+  
 
+  // index of the cell in the collection view
+  var index = 0
```

Comment:
I think making this optional would make more sense since 0 is a valid index too. WDYT

```suggestion
  var index: Int?
```